server/methods/core/packages.js

```diff
@@ -0,0 +1,44 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Packages } from "/lib/collections";
+import { Reaction } from "/server/api";
+
+/**
+ * @method updatePackage
+ * @summary updates the data stored for a certain Package.
+ * @param {String} packageName - the name of the Package to update.
+ * @param {String} field - the part of the Package's data that is to
+ * be updated.
+ * @param {Object} value - the new data that's to be stored for the said
+ * Package.
+ * @since 1.5.1
+ * @return {Object} - returns an object with info about the update operation.
+ */
+export function updatePackage(packageName, field, value) {
```

Comment:
I'll take a look at using that.

Reply:
@machikoyasuda I have considered using the method you mentioned, but I don't think it's appropriate to use it here because how it works (e.g Collection.upsert, [name.split("/").splice(-1)](https://github.com/reactioncommerce/reaction/blob/863139e3a9e66c0042e2031b99695057bba2908f/server/methods/core/registry.js#L16)) is different from how `package/update` works (Collection.update). Also, in summary, I'd have to change too many things that are beyond the scope of this ticket. In detail:
- I am using the `Form` component created by @mikemurray ([link](https://github.com/reactioncommerce/reaction/blob/folusho-convert-taxcloud-panel-to-react-3093/imports/plugins/core/ui/client/components/forms/form.js)). That component expects a `handleSubmit` function in its `props`, which it calls with  the arguments `event, changedInfo, targetField`. However, if that `handleSubmit` calls `registry/update`, it won't be able to supply the parameters expected by `registry/update`.
- I could try to change the arguments passed to `handleSubmit` by the `Form` component. But then, that would also mean that I have to change the behavior of the `handleSubmit` supplied by the `SocialSettingsContainer` component, which is another client of the `Form` component ([link](https://github.com/reactioncommerce/reaction/blob/folusho-convert-taxcloud-panel-to-react-3093/imports/plugins/included/social/client/containers/socialSettingsContainer.js)).
- I could try to change the `registry/update` too, but it has many clients, and those would require updating too.